setup.py

```diff
@@ -13,7 +13,7 @@
 AUTHOR_EMAIL = 'chris.fonnesbeck@vanderbilt.edu'
 URL = "http://github.com/pymc-devs/pymc3"
 LICENSE = "Apache License, Version 2.0"
-VERSION = "3.1"
+VERSION = "3.2"
```

Comment:
shouldn't that be 3.2rc1 here too?

Reply:
I can. Was distinguishing version vs release.